src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,87 @@
+.wrapper {
+  display: flex;
+  align-items: center;
+  height: 100vh;
+}
+
+.clock {
+  margin: 0 auto;
```

Comment:
it's wrong to use margin in bem block. Let's remove it here and add justify-content: center to the wrapper :)